.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -11,35 +11,35 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
-name: 🐛Bug
+name: Bug Report
 description: Have you got a bug? Please report it here!
+title: "[Bug]: "
+labels: ["bug", "triage"]
 projects: [ "SAFEHR-data/13" ]
```

Comment:
can remove this line as well, as we're not using the project board anymore

Reply:
sorted out here [9061750](https://github.com/SAFEHR-data/PIXL/pull/514/commits/90617507131d1366017b075312dac717425a6e68)